src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -553,6 +553,14 @@ message RpcException {
 
   // Textual message describing the exception
   string message = 2;
+
+  // Worker specifies whether exception is a user exception, 
+  // for purpose of application insights logging. Defaults to false. 
+  optional bool is_user_exception = 4;
```

Comment:
I understand these changes are already merged in Protobuf repo. Just curious, why did we skip "3" in numbering here?